api/team_invite.go

```diff
@@ -310,6 +311,12 @@ func (a *Api) AcceptMonitoringInvite(res http.ResponseWriter, req *http.Request,
 		a.sendModelAsResWithStatus(res, statusErr, http.StatusNotFound)
 		return
 	}
+	if conf.IsExpired() {
+		statusErr := &status.StatusError{Status: status.NewStatus(http.StatusUnauthorized, statusExpiredMessage)}
```

Comment:
We return a 409 but the status object is here a 401. 

```suggestion
		statusErr := &status.StatusError{Status: status.NewStatus(http.StatusConflict, statusExpiredMessage)}
```